dataset.go

```diff
@@ -38,6 +38,10 @@ func (m *DataSet) FieldByType(dsType DataSetType) (interface{}, error) {
 		return nil, ErrNilDataSet
 	}
 
+	if dsType == DataSetType_SENTIMENT {
```

Comment:
When Keyword and Sentiment are kept as separate fields in dataset.proto, this won't be needed anymore.